### PR TITLE
HERE API Interaction

### PR DESCRIPTION
**Pull request by jake-bickle**
This PR adds HERE API interaction. The client may now query a bounding box of coordinates, in which the backend will call the HERE API, gather, and parse relevant road data.

Currently, querying /api/v1/bbox will return the correct schema, but not entirely filled out. Most of the code in app.py is subject to change. In the mean time, this helps Matt out a lot.

### CODE REVIEW

File: flaskapp/road.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
+from enum import Enum
+import Levenshtein
```
Comment (rattboi) [inline flaskapp/road.py]: I don't think this is in use. Is that right?

File: flaskapp/resources/utils.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+class InvalidCoordinates(BaseException):
```
Comment (rattboi) [inline flaskapp/resources/utils.py:1]: It's a nit, but this whole file is tabbed to 6 spaces, where most others are 4 spaces.

File: flaskapp/resources/heretraffic.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import requests
```
Comment (rattboi) [inline flaskapp/resources/heretraffic.py:1]: 6 spaces indentation instead of standard 4

**Review by rattboi — DISMISSED** [review]
Not much here. This looks good to me. I add a few nits, but I'd also be fine merging as-is.

File: flaskapp/resources/heretraffic.py (hunk 3 above)
Comment (jake-bickle) [reply flaskapp/resources/heretraffic.py:1]: So I discovered my IDE will literally hide away this indentation issue! It was 4 spaces in my IDE, but in all my other text editors it was 6 lol. Fixed.

**Review by rattboi — APPROVED** [review]

**Pull request merged**